Assets/Src/Scripts/Animator/HandLandmarksController.cs

```diff
@@ -40,6 +42,8 @@ public class HandLandmarksController : MonoBehaviour {
 
     private GameObject[] _handLandmarks = new GameObject[_landmarksNum];
     private float _screenRatio = 1.0f;
+    private Transform[] _fingerTargets = new Transform[15];
```

Comment:
The mapping between this transform (15 elements) and the result from handpose model (21 landmarks) should be documented. 

I think the reason is that the handpose detects the finger tips, but those are not a joint to IK

Reply:
@superkudo Has this Henry's suggestion been solved?